pyp2rpm/pulpcore.spec

```diff
@@ -0,0 +1,105 @@
+{{ data.credit_line }}
```

Comment:
Should this use a `.j2` extension to indicate it's a template? Is that a common convention?

Reply:
We had that discussion when we added those templates in `foreman-packaging` initially: https://github.com/theforeman/foreman-packaging/pull/4318#discussion_r346842455

TL;DR: pyp2rpm would break :(